packages/grafana-toolkit/src/config/webpack/loaders.ts

```diff
@@ -90,3 +96,26 @@ export const getStyleLoaders = () => {
     },
   ];
 };
+
+export const getFileLoaders = () => {
+  const shouldExtractCss = hasThemeStylesheets();
+  const pluginJson = getPluginJson();
+
+  return [
+    {
+      test: /\.(png|jpe?g|gif|svg)$/,
+      use: [
+        {
+          loader: 'file-loader',
+          options: {
+            outputPath: 'static',
+            name: '[name].[hash:8].[ext]',
+            // When css is extracted the public path of statics is handled by MiniCssExtractPlugin.loader (see getStyleLoaders above)
+            // Otherwise the css is loaded using style-loader and we need to be specific what's the full public path for a given plugin
+            publicPath: shouldExtractCss ? undefined : `/public/plugins/${pluginJson.id}/static`,
```

Comment:
This may have weird behavior when running from the non-root path.

Reply:
yeah, still trying to figure out the right approach here.

Reply:
OK, changed the behaviour for now. The files are now inlined as base64 URIs (https://github.com/webpack-contrib/url-loader). Couldn't find an obvious/short path to solve that. The 'root_url' config option is not an issue I think as we can still have an absolute path with `public` (base tag is added to head). More problematic is `static_root_path` config option, which is know on runtime only.